websites/C/cda.pl/presence.ts

```diff
@@ -11,71 +11,95 @@ const presence = new Presence({
 
 presence.on("UpdateData", async () => {
 	const presenceData: PresenceData = {
-			largeImageKey: "logo",
+			largeImageKey: "https://i.imgur.com/AD1naGM.png",
 			startTimestamp: browsingTimestamp
 		},
-		pathname: string = document.location.pathname.toLowerCase();
+		path = document.location.pathname.toLowerCase();
```

Comment:
Please keep the variable named `pathname` instead 
```suggestion
		pathname = document.location.pathname.toLowerCase();
```